packages/dds/tree/src/changeset/toDelta.ts

```diff
@@ -13,8 +13,11 @@ import { ProtoNode, Transposed as T } from "./format";
  * @param changeset - The Changeset to convert
  * @returns A Delta for applying the changes described in the given Changeset.
  */
-export function toDelta(changeset: T.Changeset): Delta.Root {
-    return convertMarkList<Delta.OuterMark>(changeset.marks);
+export function toDelta(changeset: T.LocalChangeset): Delta.Root {
+    // Save result to a constant to work around linter bug:
```

Comment:
if we explicitly use JsonableTree instead of PlaceholderTree, I think it should avoid this linter bug.

Reply:
Oh, I guess you have a different recursive type here. Might be useful to apply the same trick though: make an interface extending the recursive type and it generally works a lot better.